sklearn/datasets/_base.py

```diff
@@ -765,6 +765,11 @@ def load_diabetes(return_X_y=False, as_frame=False):
 
         .. versionadded:: 0.23
 
+    standardized : bool, default=True
+        If True, the feature variables are mean centered and scaled by the
+        standard deviation times `n_samples`.
```

Comment:
So this should be "standard deviation times the square root of `n_samples`.", no?

This is weird to call this "standardized" because it does not not match the usual Standardization we implement in `StandardScaler`.

Maybe we should rename this `standardized` parameter to a more neutral name such as `preprocessed` or `scaled`.

Reply:
Good point. I changed the description to reflect `standard deviation times the square root of n_samples` and renamed the parameter to `scaled`.

Reply:
Just realized that I took the description from `descr/diabetes.rst`, so I updated that one too